src/brain/operating_unit_recorder.cpp

```diff
@@ -222,7 +222,14 @@ void OperatingUnitRecorder::AggregateFeatures(brain::ExecutionOperatingUnitType
     }
```

Comment:
We also need to check that the pipeline ids are the same before we merge features here, since now we differentiate the multiplication by the pipeline ids.

Reply:
Added a check on the set of translators.